src/Parser/Priest/Discipline/Normalizers/AtonementSuccessiveDamage.js

```diff
@@ -0,0 +1,45 @@
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+import isAtonement from '../Modules/Core/isAtonement';
+
+class AtonementSuccessiveDamage extends EventsNormalizer {
+
+  normalize(events) {
+    const fixedEvents = [];
+    const _damageEventIndexes = [];
+
+    let _encounteredTargetIDs = [];
+
+    events.forEach((event, eventIndex) => {
+
+      fixedEvents.push(event);
+
+      if(event.type === "damage" && event.sourceIsFriendly && !event.targetIsFriendly){
+        _damageEventIndexes.push(eventIndex);
+        _encounteredTargetIDs = [];
+        return;
+      }
+
+      if(event.type === "heal" && isAtonement(event)) {
+
+        // We encountered a targetID we already encountered since the last damage
+        // event. We push down the last damage event here
+        if(_encounteredTargetIDs.indexOf(event.targetID) >= 0) {
+          const lastDamageEvent = fixedEvents.splice(_damageEventIndexes[_damageEventIndexes.length -1], 1)[0];
+          fixedEvents.splice(fixedEvents.length - 1, 0, lastDamageEvent);
+          _encounteredTargetIDs = [];
+          return;
+        }
+
+        //  We ignore atonement on self in the handling of repeating targetIDs
+        //  Because of latency issues
```

Comment:
I think this "because of" is confusing and perhaps even misleading. How about stating it's being handled separately in another normalizer?